services/nextjs-app/src/app/globals.css

```diff
@@ -0,0 +1,27 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+:root {
```

Comment:
It might be nice to pop it into a css folder, just so things are structured nicely? Will also help if we decide to create a second css file or something!

Reply:
Ooh yeah good idea, i'll add a styles folder